testing/ios/IosUnitTests/run_tests.sh

```diff
@@ -8,9 +8,6 @@ if [ $# -eq 1 ]; then
   FLUTTER_ENGINE=$1
 fi
 
-set -o pipefail && xcodebuild -sdk iphonesimulator \
```

Comment:
`set -o pipefail` was the magic that worked with xcpretty.  When we removed xcpretty it was no longer magical.

Reply:
fyi @jmagman

Reply:
It wasn't being used in the python script, and I couldn't seem to get it working in a way the script liked. Honestly, I'm not sure its worth it. Every time I've ever had a test failure I had to re-run my tests without xcpretty to figure out why.

(@xster same reply to your comment above)

Reply:
I'm fine removing xcpretty, just wanted to explain why this was there and how this broke.